language/compiler/bytecode-source-map/tests/dummies.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use bytecode_source_map::mapping::SourceMapping;
+use move_binary_format::{binary_views::BinaryIndexedView, file_format::CompiledScript};
+use move_ir_types::location::Spanned;
+
+static EMPTY_SCRIPT: &[u8] = include_bytes!("../../../../types/src/test_helpers/empty_script.mv");
```

Comment:
This seems like it could cause issues/introduces a non-obvious Diem dependency. Could you instead copy this empty script bytes over into this directory and use that?